static/js/stream_popover.js

```diff
@@ -389,6 +390,14 @@ export function register_click_handlers() {
 }
 
 export function register_stream_handlers() {
+    // New topic in stream menu
+    $("body").on("click", ".new_topic_stream", (e) => {
+        const sub = stream_popover_sub(e);
+
+        hide_stream_popover();
+        compose_actions.start("stream", {trigger: "new topic button", stream: sub.name});
+    });
```

Comment:
This should do `e.stopPropagation`, like other normal click handlers.

Reply:
I have added this. And really this was the most needed line in the code which I was mising.